Managers/NetworkServer.h

```diff
@@ -239,22 +238,26 @@ namespace RTE {
 
 		bool m_SendEven[c_MaxClients]; //!<
 
-		bool m_ShowStats; //!<
-		bool m_ShowInput; //!<
-
 		bool m_SendSceneSetupData[c_MaxClients]; //!<
 		bool m_SendSceneData[c_MaxClients]; //!<
 		bool m_SceneAvailable[c_MaxClients]; //!<
 		bool m_SendFrameData[c_MaxClients]; //!<
+
 		std::mutex m_SceneLock[c_MaxClients]; //!<
 
-		unsigned char m_TerrainChangeBuffer[c_MaxClients][c_MaxPixelLineBufferSize]; //!<
+		unsigned char m_PixelLineBuffer[c_MaxClients][c_MaxPixelLineBufferSize]; //!< Buffer to store currently transferred pixel data line.
+		unsigned char m_PixelLineBufferDelta[c_MaxClients][c_MaxPixelLineBufferSize]; //!< Buffer to store currently transferred pixel data line.
+
+		unsigned char m_CompressedLineBuffer[c_MaxClients][c_MaxPixelLineBufferSize]; //!< Buffer to store compressed pixel data line
+
+		unsigned char *m_PixelLineBuffersPrev[c_MaxClients];
```

Comment:
These are, presumably, the versions of the above buffers, just from the last update? Or maybe the last time stuff was sent, I dunno, haven't read the cpp yet.

Reply:
\_/¯( ͜。 ͡ʖ ͜。)¯\\\_